v22.1/take-and-restore-locality-aware-backups.md

```diff
@@ -40,11 +40,13 @@ You can restore the backup by running:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> RESTORE FROM '{path/to/backup/subdirectory}' IN ('s3://us-east-bucket', 's3://us-west-bucket');
+> RESTORE FROM LATEST IN ('s3://us-east-bucket', 's3://us-west-bucket');
 ~~~
 
 Note that the first URI in the list has to be the URI specified as the `default` URI when the backup was created. If you have moved your backups to a different location since the backup was originally taken, the first URI must be the new location of the files originally written to the `default` location.
 
+To restore from a specific backup, use [`RESTORE FROM {subdirectory} IN ...`](restore.html#restore-a-specific-backup)
```

Comment:
i think this page could use some examples which use the incremental_location parameter, which also takes locality aware URIs. That change could also wait for another pre 22.1 pr, as I['m still figuring out](https://github.com/cockroachdb/cockroach/pull/79121) locality aware SHOW BACKUP.

Reply:
I will cover this in a follow-up PR, with a separate docs issue that should hit my triage soon